tests/configs/match-final

```diff
@@ -0,0 +1,11 @@
+Host jump
+    HostName jump.example.orig
+    Port 1003
+
+Host finally
+    HostName finally.example.org
+    Port 1001
+
+Match final host "*.example.org" !host jump.example.org
+    ProxyJump jump
+    Port 1002
```

Comment:
Coming back to this, looking more closely at the changes.

Unless I'm missing something, there's no test case that finds this entry. Is that intentional? Should this entry never be reachable?

(I don't know how `Match final` is supposed to work, so this may be a silly question.)

If there is a case that would be able to reach this entry (looks like any arbitrary subdomain other than `jump` or `finally`?), it would be worth adding.

Reply:
Thank you _very much_ for your patience on this, btw.

Reply:
It's basically "does not use the default port from the match final" for the hosts in `test_final`

Add a host without port so we can test positive as well?